akka-docs/.history

```diff
@@ -1 +0,0 @@
-exit
```

Comment:
hm that's a bash thing, it should be deleted entirely. I wonder how it ended up here